tests/unit/backend/layers/business/test_business.py

```diff
@@ -775,6 +775,9 @@ def test_get_dataset_artifacts_ok(self):
         self.assertCountEqual(
             [a.type for a in artifacts], [DatasetArtifactType.H5AD, DatasetArtifactType.CXG, DatasetArtifactType.RDS]
         )
+        self.assertCountEqual(
+            [a.get_file_name() for a in artifacts], ["local.h5ad", "local.cxg", "local.rds"]
```

Comment:
Should we also check for equality of values instead of just number of files?

Reply:
Turns out that `assertCountEqual` actually [checks if two iterables have the same elements regardless of order](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual), not just the count.